test/csi-driver.feature

```diff
@@ -0,0 +1,39 @@
+Feature: SharedSecrets and SharedConfigMap
+
+    As a user of openshift tech preview feature
```

Comment:
5 high level questions @prietyc123 @jitendar-singh 

1)  do you all envision descriptions like these being a replacement for what QE traditionally documented in polarion, or do you see these descriptions ultimately being items you drop into place into polarion? 

2) I am assuming we will see you all create separate `Feature` items for all the different scenarios that QA should cover for shared resources and 4.10.  Is that correct?

3) If so, do you all plan to track all of that under BUILD-256?

4) If so, can we expect separate PRs for each of the scenarios as you tackle them?

5) For items that have to be verified manually, like the metrics stuff through the console, should we expect those items to be described in polarion and presented to development for review?

@openshift/openshift-team-build-api FYI

Reply:
@gabemontero these description will replace polarion description that we do manually currently and then dev review that. Once we have these description ready, we have automated script in place to pull data to polarion. 
After the changes are done, team will review it in github itself. No need to go for polarion and stuff. 

> If so, can we expect separate PRs for each of the scenarios as you tackle them?

I will open separate pr for each and every e2e csi driver scenarios.

Reply:
OK @prietyc123 thanks for those clarifications.  So, next steps from my point of view:
- I have already asked @jitendar-singh to capture the answer to my questions in his #80 which he has told me will serve as the base for this PR.  I want our new / burgeoning process here documented in the repo's README, not just PR comment or slack comments ... he has updates in that PR which I'll look at shortly.  I'll cross reference as needed between what you noted here and his updates.
- separate PRs for each scenario is certainly reasonable; but we'll need to track somehow how far along you are in crafting items for each of the scenarios, so that we can properly tag items in Jira as "tc-approved".  How would you like to track ?  Minimally, a central list of the scenarios you will be going after seems prudent.

@adambkaplan FYI